### DISCUSSION TITLE
Call for Mentors for Open Source Promotion Plan 2022!

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#4317] [General]
@apolloconfig/committers @apolloconfig/members 

Apollo community has signed up for [Open Source Promotion Plan](https://summer-ospp.ac.cn/), which encourages university students to participate in the development and maintenance of open-source software. Students can apply for the project they are interested in and guided by the senior maintainer (community mentor) once they are selected.

This is a great opportunity to cultivate fresh blood in the open-source community, and I strongly encourage committers and members to sign up for the mentor and contribute tasks for the students to participate.
If you want to be a mentor but don't have a task in your mind, then you may use choose from the following candidates:
* #4079
* #4294 
* #4315
* #4316

Here is the timeline:

| **Date** | **Event** |
| ----------- | ---------------------- |
| 03/28-04/15 | Community Registration |
| 04/16-04/20 | Project Number Communication |
| 04/21-05/10 | Project Release and Committee Review |
| 04/21-05/20 | Communication |
| 05/21-06/04 | Students Submit Application |
| 06/05-06/11 | Project Application Review (Mentor) |
| 06/12-06/14 | Project Application Review (Committee) |
| 06/15 | Accepted Student Projects Announced |
| 06/16-06/30 | Project Warm-up |
| 07/01-09/30 | Development |
| 10/01-10/31 | Final Term Review |
| Early November | Announcement of the Final Results |

**Comment by Anilople** [comment]
+1

---

How to be the mentor, just reply `+1` under this discussion?

**Reply by nobodyiam** [reply]
+1 with a link to the task you want to guide the student.
If you don't have a task in your mind, then you could choose from the candidates listed in the thread. 😀

**Reply by Anilople** [reply]
+1 #3208

---

I want to resolve the clear of config's release table, i.e #3208

**Comment by klboke** [comment]
+1  

- If there are no other mentors to sign up, you can leave #4079 to me, or to other mentors if you have one, because I already participated last year

**Comment by lepdou** [comment]
+1

- If there are no other mentors to sign up, you can leave https://github.com/apolloconfig/apollo/issues/4315 to me, or to other mentors if you have one, because I already participated last year

**Comment by nobodyiam** [comment]
+1 with
- https://github.com/apolloconfig/apollo/issues/4294

**Comment by nobodyiam** [comment]
@Anilople @klboke @lepdou 

I created a [wiki page](https://github.com/apolloconfig/apollo/wiki/Open-Source-Promotion-Plan-Summer-2022-Tasks) for the Open Source Promotion Plan project, please help to fill in the title, description, difficulties, etc of your task so that I could submit them to the organizers.
